go/obscuronode/nodecommon/types.go

```diff
@@ -21,6 +21,7 @@ type (
 	EncryptedTransactions []byte // A blob of encrypted transactions, as they're stored in the rollup.
 	EncryptedParams       []byte // The params for an off-chain call or getBalance request, as a JSON list encrypted with the public key of the enclave.
```

Comment:
we should create some structs for each message that is being sent. We can marshall it to json, but we should have structs. Quite hard to follow without them.
Also, you could create "EncryptedXXX []byte" for each of them. This way we could follow the logic better.

Reply:
I think I've addressed this. Can you check?